### ISSUE TITLE
Implement "special" rounds

### ISSUE THREAD

**Issue by AnotherKamila** [netsec-ethz/scionlab-workshop#19]
A few rounds should be pre-announced and weighed much more than the "default", so that people can submit their "best" code for them.

There should be enough of them for randomising stuff, but not so many that a contestant will be unable to catch up if they write good code very late.

**Comment by AnotherKamila** [comment]
There are 2 parts of this task:
1. The webserver needs to know which rounds are special, and needs to multiply the scores for those before submitting.
2. We need to let the users know which rounds are special => needs to be in the dashboard.

For 2, I think it makes sense to have a countdown on the dashboard, i.e. a big thingy saying "next special round in X min".

One option is that we write all the special round numbers into influxdb at the beginning of the contest and we write a fancy query to tell us "(next special round - curr_round) * 60s"; another option is that we just have the final value in influx and we update it on every submit.
@mkowalski and @giacgiuliari pick something :D

**Comment by mkowalski** [comment]
There is no such a thingy as "update in Influx". If you write you write, if you want to overwrite, you write a next one with a new timestamp and read the one with the latest.

**Comment by mkowalski** [comment]
- `write all the special round numbers into influxdb at the beginning of the contest` is no go; this is time-series, not a relational DB; how would you timestamp the point?
- `we just have the final value in influx and we update it on every submit` I will try simply assuming no prior knowledge how time-series store data
  - for any point in time I want to be able to ask a question "what is the next special round" and every time I ask I need to get only one answer, think one string or one number
  - given the time-based nature of the question, question "give me all the existing rounds" is highly inappropriate; possible using _schema exploration_, but still inapropriate
  - long story short, every point (i.e. a single record) represents state of the universe for a timestamp it contains; a influx query gives you a possibility to retrospectively ask for a state of the universe at a selected time or over a period of time

**Comment by AnotherKamila** [comment]
Okay, thanks for the explanation. 

@giacgiuliari Since this is the case, it's probably simplest if you push `next_special_round` into influxdb with the other stuff (i.e. same as the `round_as_field` thing).

**Comment by mkowalski** [comment]
I could also give you a simple web interface with a button to publish the next special round, kinda “manual drive” mode. The value does not have to be pushed continuously (but of course can), it’s just the latest should always reflect only the next (i.e. the closest) round

I started already drafting something so probably best if I sync with Giacomo
________________________________

**Comment by AnotherKamila** [comment]
All done in the dashboard, yay!
